docs/advanced/subscriptions.md

```diff
@@ -69,6 +69,45 @@ we return to the `subscriptionExchange` inside `forwardSubscription`.
 
 [Read more about `subscription-transport-ws` on its README.](https://github.com/apollographql/subscriptions-transport-ws/blob/master/README.md)
 
+
+## Setting up `graphql-ws`
+
+If your GraphQL API server is using [graphql-ws](https://github.com/enisdenjo/graphql-ws),
+you'll be able to use it here too!
+
+```js
+import { createClient, defaultExchanges, subscriptionExchange } from 'urql';
+import { createClient as createGraphqlWsClient } from 'graphql-ws';
```

Comment:
nit: I'd just name this `createWSClient` or `createSubscriptionClient` since it's all equivalent but more easily readable; basically because the casing on the import is a little inconsistent otherwise

Reply:
[I settled with `createWSClient`.](https://github.com/FormidableLabs/urql/pull/1118/commits/d37ca3f45aea54b59422ccebe334f120dd1e56d1)

Mainly because you can transport queries and mutations over `graphql-ws` too, but not only that - you can also stream executions for supporting directives like `@defer` or `@stream`.